### PR TITLE
Add course-materials route

### LINKED ISSUE #1665: Provide a consolidated view of all LMs for a given course
In order to expedite student study experience, provide a display similar to (or perhaps a version of) the course summary view, which displays all learning materials for a course, organized by session / date descending.

### PR DESCRIPTION
**Pull request by jrjohnson**
Displays all the learning materials associated with a course including
all of those attached to sessions.

Fixes #1665

WIP:
- [x] Link from course overview
- [x] Improved summary and header
- [x] link back to course
- [x] Add missing translations
- [x] Fix test failures on travis (they pass locally)

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 21f64dec6a58ea34d9a8e2088ad8854fdff566bb

https://deploy-preview-2246--ilios-frontend.netlify.com

**Comment by jrjohnson** [comment]
@saschaben this is ready for review again.  I took some liberties with the header as I'm hoping to re-use it on top of some other course pages where we currently have a non-edit header.

**Comment by saschaben** [comment]
@jrjohnson this looks really good. Two things leap out at me: one, we need a bit more padding under the course title in the header; and when we load this summary page, the header should _not_ include the `fa-archive` link (or we should grey it out). Otherwise this looks like a winning header, and I think this pattern is one we can use generally.

One other thing: now that we have this awesome list, it is clear that we need to extend the `learning_material.title` attribute from the current 60 characters to something larger, like 255.

File: app/routes/course-materials.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+const { Route, RSVP } = Ember;
+const { Promise, all, map } = RSVP;
+
+export default Route.extend(AuthenticatedRouteMixin, {
+  afterModel(course){
+    return all([
+      this.loadCourseLearhingMaterials(course),
```
Comment (stopfstedt) [inline app/routes/course-materials.js]: Learning, not Learhing. Please fix this everywhere.

Comment (jrjohnson) [inline app/routes/course-materials.js]: DAMN!  Even for me that is a big typo.

Comment (jrjohnson) [inline app/routes/course-materials.js]: Ok fixed.  Ready for review again.

**Pull request merged**